src/Database/Driver/Mysql.php

```diff
@@ -157,6 +160,28 @@ public function connect(): void
         }
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function run(Query $query): StatementInterface
+    {
+        $statement = $this->prepare($query);
+        $query->getValueBinder()->attachTo($statement);
+
+        if ($query instanceof SelectQuery) {
+            try {
+                $this->getPdo()->setAttribute(PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, $query->isBufferedResultsEnabled());
```

Comment:
I believe we did the the same in 4.x, so this should be ok. I'm wondering if setting the attribute per query is necessary though.

Reply:
Yes it's the same as 4.x, no idea about the performance implications.